lighthouse-core/audits/load-fast-enough-for-pwa.js

```diff
@@ -11,17 +11,14 @@
  *  Afterwards, we report if the TTFI is less than 10 seconds.
  */
 
+const isDeepEqual = require('lodash.isequal');
 const Audit = require('./audit');
-const URL = require('../lib/url-shim');
-const targetLatencyMs = require('../config/constants').throttling.mobile3G.rttMs;
-const Sentry = require('../lib/sentry');
+const mobile3GThrottling = require('../config/constants').throttling.mobile3G;
 const Util = require('../report/v2/renderer/util.js');
 
-// Maximum TTFI to be considered "fast" for PWA baseline checklist
+// Maximum TTI to be considered "fast" for PWA baseline checklist
 //   https://developers.google.com/web/progressive-web-apps/checklist
-const MAXIMUM_TTFI = 10 * 1000;
-
-const WHITELISTED_STATUS_CODES = [307];
+const MAXIMUM_TTI = 10 * 1000;
```

Comment:
Just to call out explicitly: 
We're changing the PWA speed threshold from `TTFI <= 10s` to be a slightly more aggressive `TTI < 10s`.